contentcuration/contentcuration/frontend/shared/mixins.js

```diff
@@ -278,19 +279,21 @@ export function generateSearchMixin(filterMap) {
         this.navigate({});
       },
       navigate(params) {
-        this.$router
-          .replace({
-            ...this.$route,
-            query: {
-              ...params,
-              page: 1, // Make sure we're on page 1 for every new query
-            },
-          })
-          .catch(error => {
-            if (error && error.name != 'NavigationDuplicated') {
-              throw error;
-            }
-          });
+        if (!isEqual(this.$route.query, params)) {
+          this.$router
+            .replace({
+              ...this.$route,
+              query: {
+                ...params,
+                page: 1, // Make sure we're on page 1 for every new query
+              },
+            })
+            .catch(error => {
+              if (error && error.name != 'NavigationDuplicated') {
```

Comment:
Is there a reason to ignore `NavigationDuplicated` errors?

Reply:
They might have been happening frequently, although I don't remember when this was added in